src/components/projects/projectId/attributes/attributeId/AttributeCalculations.tsx

```diff
@@ -385,7 +390,7 @@ export default function AttributeCalculation() {
                         {usableAttributes.length == 0 && <div className="text-sm font-normal text-gray-500">No usable attributes.</div>}
                         {usableAttributes.map((attribute: Attribute) => (
                             <Tooltip key={attribute.id} content={attribute.dataTypeName + ' - ' + TOOLTIPS_DICT.GENERAL.CLICK_TO_COPY} color="invert" placement="top">
-                                <span onClick={() => copyToClipboard(attribute.name)}>
+                                <span onClick={copyToClipboardFunc(attribute.name)}>
```

Comment:
this doesn't work since the are setting a call of a function as the reference. we need to set function ()=> instead

Reply:
this is moved as part of the callback `useCallback(() => () => {...},[])`, i tested the copy and it worked or  something else is meant?